### DISCUSSION TITLE
FalixNodes successfully detected UC-Driver

### DISCUSSION THREAD

**Discussion by xAffan** [ultrafunkamsterdam/undetected-chromedriver#1320] [Q&A]
The website will redirect if it detects that something is wrong.

Code to reproduce the issue.
```import undetected_chromedriver as uc
import time

driver = uc.Chrome()
driver.get('https://falix.gg/?id=12345678')
while True:
   # The website does not redirect to about:blank if the browser is undetected
   if driver.current_url == "about:blank":
        print('Browser detected!')
        break
   else:
        time.sleep(2)
```
        
Visit the same url normally = no such issue.

**Comment by xAffan** [comment]
I would like to add that the website detects dev tools and redirects you. I think detecting dev tools correlates to detecting UC - Driver.

**Comment by sahariar-safin** [comment]


**Reply by xAffan** [reply]
Bro that is chatGPT answer. That website is detecting devtools and there is no way around it until the release of driverless UC
